app/bundles/EmailBundle/Helper/MailHelper.php

```diff
@@ -1230,10 +1230,16 @@ public function setEmail(Email $email, $allowBcc = true, $slots = [], $assetAtta
 
         $this->email = $email;
 
+        $fromEmail      = $email->getFromAddress() ?? $this->coreParametersHelper->get('mailer_from_email', '');
+        $fromName       = $email->getFromName() ?? $this->coreParametersHelper->get('mailer_from_name', '');
+        $replyToAddress = $email->getReplyToAddress() ?? $this->coreParametersHelper->get('mailer_reply_to_email', '');
+        $this->setFrom($fromEmail, $fromName);
+        $this->setReplyTo($replyToAddress, '');
+
         $subject = $email->getSubject();
 
         // Set message settings from the email
-        $this->setSubject($subject);
+        $this->setSubject($subject) ?? '';
```

Comment:
I guess you meant 
```suggestion
        $this->setSubject($subject ?? '');
```
But even that wouldn't be OK to do just to make the unit tests pass. I'd make sure that the subject is set in the Email entity in the test and if the Email entity is mocked then the `getSubject` method must be configured on the mock to return a string. I'm not sure why the tests fail this way though. I don't see any change in subject here.